### PR TITLE
[RAPPS] improve when to call  ConInitStdStreams

### PR DESCRIPTION
**Pull request by kernelbin**
## Purpose

trivial improvement about when to call ConInitStdStreams

### CODE REVIEW

**Comment by HBelusca** [comment]
Hmmm....
Wouldn't it be possible to have some "cmd_main" or "console_main" (whatever...) function that is called when RAPPS in text(command)-mode is triggered, that would do this ConInit thing once ?

**Comment by kernelbin** [comment]
> Hmmm....
> Wouldn't it be possible to have some "cmd_main" or "console_main" (whatever...) function that is called when RAPPS in text(command)-mode is triggered, that would do this ConInit thing once ?

Not until we parsed the cmdline did we know it's going to start in CLI or not🤣
so having a ```console_main``` or whatever means I need to parse cmdline twice, or passing a lot of information retrieved from cmdline to ```console_main``` ...
It's possible but IMO it's possibly worse than current desing🤣

File: base/applications/rapps/unattended.cpp [hunk 1]
```diff
@@ -283,18 +283,22 @@ BOOL ParseCmdAndExecute(LPWSTR lpCmdLine, BOOL bIsFirstLaunch, int nCmdShow)
     }
     else if (MatchCmdOption(argv[1], CMD_KEY_FIND))
```
Comment (learn-more) [inline base/applications/rapps/unattended.cpp:284]:
```suggestion
    ConInitStdStreams();
    if (MatchCmdOption(argv[1], CMD_KEY_FIND))
```
You'd also need to add a return in the first if() case

Comment (learn-more) [reply base/applications/rapps/unattended.cpp:284]: then you can remove the init from all

**Pull request merged**